src/main/java/service/imp/ReaderServiceImpl.java

```diff
@@ -0,0 +1,22 @@
+package service.imp;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import service.ReaderService;
+
+public class ReaderServiceImpl implements ReaderService {
+    private List<String> dataFromFile = new ArrayList<>();
+
+    @Override
+    public List<String> read(String fileName) {
+        try {
+            dataFromFile = Files.readAllLines(Path.of(fileName));
+        } catch (IOException e) {
+            throw new RuntimeException("Can't find data with this path:  " + fileName);
```

Comment:
When catching the `IOException`, it's better practice to include the original exception when throwing a new RuntimeException. This way, you're preserving the stack trace which can be invaluable for debugging issues. Use `throw new RuntimeException("Can't read data from file: " + fileName, e);`